_docs/advanced-features.md

```diff
@@ -115,23 +115,25 @@ The latter is `~/.dockstore/cache/` by default and can be set to any directory l
 
 By default, Dockstore will attempt to download files up to three times. Control this with the `file-provision-retries` parameter inside `~/.dockstore/config`.
 
-## Running CWL-runner with extra tags
+## Running workflows with extra flags
```

Comment:
My last comment was confusing... The parameter only applies to CWL, but it applies equally to the 3 CWL tools we support, cwtool, cwlrunner and Cromwell. And I believe it also applies to tools as well as workflows.

So... how about something like `Running CWL with extra flags`?